backend/tests/controllers/test_app.py

```diff
@@ -0,0 +1,415 @@
+import unittest
```

Comment:
How much percent test coverage is this?

Reply:
its around 52%  right now and overall it is 58%